test/main.test.js

```diff
@@ -0,0 +1,308 @@
+});
+
+test('uploadFeed(files) - other status codes', async () => {
+    expect.assertions(1);
+    const port = await createTestServer([
+        {
+            verb: 'post',
+            path: '/feed',
+            cb: (req, res) => res.status(300).send('Bad request!'),
+        },
+    ]);
+    const fakeFiles = [];
+    const fakeOptions = {};
+    const client = new Client({ buildServerUri: `http://127.0.0.1:${port}` });
+
+    const result = client.uploadFeed(fakeFiles, fakeOptions);
+
+    await expect(result).rejects.toEqual(new Error('Asset build server responded with unknown error. Http status 300'));
+});
+
+test.skip('uploadFeed(files) - css');
+
+test.skip('createRemoteBundle(sources) - request error', async () => {
+    expect.hasAssertions();
+    const port = await createTestServer([
+        {
+            verb: 'post',
+            path: '/bundle',
+            cb: (req, res, next) => next(new Error('Error!')),
+        },
+    ]);
+    const fakeSources = ['a12das3d', '12da321fd'];
+
+    const client = new Client({ buildServerUri: `http://127.0.0.1:${port}` });
+    const result = client.createRemoteBundle(fakeSources);
+
+    await expect(result).rejects.toBe('Error!');
+});
+
+test('createRemoteBundle(sources) - 200', async () => {
+    expect.assertions(1);
+    const port = await createTestServer([
+        {
+            verb: 'post',
+            path: '/bundle',
+            cb: (req, res) => res.send('success!'),
+        },
+    ]);
+    const fakeSources = ['a12das3d', '12da321fd'];
+
+    const client = new Client({ buildServerUri: `http://127.0.0.1:${port}` });
+    const result = client.createRemoteBundle(fakeSources);
+
+    await expect(result).resolves.toBe('success!');
+});
+
+test('createRemoteBundle(sources) - 202', async () => {
+    expect.assertions(1);
+    const port = await createTestServer([
+        {
+            verb: 'post',
+            path: '/bundle',
+            cb: (req, res) => res.status(202).send('success 202'),
+        },
+    ]);
+    const fakeSources = ['a12das3d', '12da321fd'];
+
+    const client = new Client({ buildServerUri: `http://127.0.0.1:${port}` });
+    const result = client.createRemoteBundle(fakeSources);
+
+    await expect(result).resolves.toBe('success 202');
+});
+
+test('createRemoteBundle(sources) - 400', async () => {
+    expect.assertions(1);
+    const port = await createTestServer([
+        {
+            verb: 'post',
+            path: '/bundle',
+            cb: (req, res) => res.status(400).send({ message: 'Bad request' }),
+        },
+    ]);
+    const fakeSources = ['a12das3d', '12da321fd'];
+
+    const client = new Client({ buildServerUri: `http://127.0.0.1:${port}` });
+    const result = client.createRemoteBundle(fakeSources);
+
+    await expect(result).rejects.toEqual(new Error('Bad request'));
+});
+
+test('createRemoteBundle(sources) - other status codes', async () => {
+    expect.assertions(1);
```

Comment:
No need when there are no code paths that may be unreachable.
Doesn't hurt, of course